aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -1080,7 +1081,7 @@ void DimCounter::increment(const std::array<int64_t, 2>& step) {
 }
 
 std::array<int64_t, 2> DimCounter::max_2d_step() const {
-  int64_t step0 = std::min(shape[0] - values[0], range.end - offset);
+  int64_t step0 = std::min(shape[0] - values[0], range.end - offset); 
```

Comment:
hmmm these seem not really changing anything? maybe checkout from master so it's not in the changeset?

Reply:
There is one blank line 806. I have fixed this issue.